docker-compose.yml

```diff
@@ -31,5 +34,15 @@ console:
     - ./docker/conf/php.ini:/usr/local/etc/php/php.ini:ro
   links:
     - nginx
-  env_file:
-    - docker-compose.env
+  environment:
+    - UNIX_USERNAME
+    - UNIX_GROUP
+    - UNIX_UID
+    - UNIX_GID
+
+composer:
+  image: composer
```

Comment:
We already have composer and plugins we need on the php-dev image.

Reply:
Yeah but how are you supposed to run `composer install`? It's not documented…

Reply:
`"echo "Run 'composer install' command to continue."`

On the script you deleted. :-)

I agree it's not very efficient, remember I did it very quickly at start. #DevExcuse :stuck_out_tongue:

Reply:
Fixed